src/lib/jsonParse/jsonParse.cpp

```diff
@@ -414,10 +414,9 @@ static std::string jsonParse
     if (ciP->answer.empty())
     {
       ciP->answer = std::string("JSON Parse Error: unknown field: ") + path.c_str();
```

Comment:
Makes sense split the answer? (similar as described above)

Reply:
In this case I think is not possible, as ciP->answer is modified here only sometimes (it is within "if")

Reply:
NTC